scripts/utils/render.js

```diff
@@ -27,19 +28,22 @@ const projectsCardBuilder = (projectsArr)=> {
 // projects form
 
 const overviewCardBuilder = (reposArray) => {
-  const pinnedRepos = reposArray.filter((r) => r.isPinned);
+  const pinnedRepos = reposArray
+    .map((r, index) => ({...r, index}))
+    .filter((r) => r.isPinned);
 
   let domString = `
     <div class="p-3">
       <h2>Pinned</h2>
-      <div class="d-flex flex-wrap">
+      <div class="d-flex flex-wrap" id="pinnedCards">
         ${pinnedRepos
           .map(
             (repo) => `
         <div class="card bkg-dark gh-border-dark text-white m-1" style="width: 18rem;">
           <div class="card-body">
             <h5 class="card-title">${repo.name}</h5>
             <p class="card-text">${repo.description}</p>
+            <button class="btn btn-success" id="unpinRepo---${repo.index}">Unpin</button>
```

Comment:
Added an Unpin button here with an id that contains the words unpin and the index of the repo that I can reference in the event that happens on click